Source/DafnyLanguageServer/Handlers/Custom/VerificationHandler.cs

```diff
@@ -43,13 +43,12 @@ public class VerificationHandler : IJsonRpcRequestHandler<VerificationParams, bo
     var translatedDocument = await documentEntry.TranslatedDocument;
     var requestPosition = request.Position;
     var tasks = GetTasksAtPosition(translatedDocument, requestPosition).ToList();
-    var failedTaskRuns = 0;
     foreach (var taskToRun in tasks) {
       var verifiedDocuments = documentLoader.Verify(translatedDocument, taskToRun, CancellationToken.None);
       documentEntry.Observe(verifiedDocuments);
     }
 
-    return failedTaskRuns != tasks.Count;
+    return !documentEntry.Idle;
```

Comment:
Won't this return true immediately because verification is running in a separate thread? What is the purpose of returning a value which might or might not be true?

Reply:
Honestly I need to do more testing on the VSCode side to see what exactly needs to be returned here.

The current approximation is to return true if more notifications are incoming, and false otherwise. If false is returned, the extension won't start a 'test run' since no notifications will arrive to finish the run.